geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+env.mako_tmpl

```diff
@@ -29,7 +29,7 @@ TILEGENERATION_SQS_QUEUE=<queue_name>
 GUNICORN_PARAMS=--bind=:8080 --worker-class=gthread --threads=10 --workers=5 --timeout=60 --max-requests=1000 --max-requests-jitter=100
 TINYOWS_URL=http://tinyows:8080/
 MAPSERVER_URL=http://mapserver:8080/
-PRINT_URL=http://print:8080/print/
+PRINT_URL=https://mutualized-print.paas-ch-3.camptocamp.com/print/demo_geomapfish_24/
```

Comment:
Are you sure you want to hardcode the demo_geomapfish_24 app?

Reply:
Thanks, fixed :-)